app/Models/GearReminder.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+
+class GearReminder extends Model
+{
+    use HasFactory;
```

Comment:
HasUuids is missing here (although I'll soon realise when saving models fails anyway 🙃)